student-test/assignment1/Test_validate_username.c

```diff
@@ -17,6 +17,8 @@ void test_validate_my_username()
     /**
      * TODO: Replace the line below with your code here as described above to verify your /conf/username.txt 
      * config file and my_username() functions are setup properly
-     */
-    TEST_ASSERT_TRUE_MESSAGE(false,"AESD students, please fix me!");
+     */ 
+	const char *username;
+	username = my_username();
+	TEST_ASSERT_EQUAL_STRING_MESSAGE("parthkharade",username,"Username Mismatch");
```

Comment:
@parthkharade you need to use the `malloc_user_name_from_conf_file` function to read your name from conf file instead of hard coding here.